modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -105,7 +105,7 @@ $ oc adm upgrade --to-latest
 +
 [source,terminal]
 ----
-$ oc adm upgrade
```

Comment:
I think `oc adm upgrade` is what we recommend user to use. And we can see the version from `oc adm upgrade` output, as well as upgrade progress.

Reply:
@shellyyang1989 okay. Would you be able to help me out with the output?

Reply:
Let's see Dev's opinion:)

Reply:
Yeah I think `oc adm upgrade` is preferrable from our PoV, we want users to use that in most cases. Its output will look something like this during the upgrade, consistent with what the web console shows:

```console
$ oc adm upgrade
info: An upgrade is in progress. Working towards 4.13.0-rc.3: 106 of 841 done (12% complete), waiting on etcd, kube-apiserver
...
```

I only had a 4.12 nightly cluster at hand which shows some undersirable output there, I'll get you an output from a proper version upgrade.

Reply:
```console
$ oc adm upgrade
info: An upgrade is in progress. Working towards 4.12.11: 104 of 830 done (12% complete), waiting on etcd, kube-apiserver

Upstream is unset, so the cluster will use an appropriate default.
Channel: fast-4.12 (available channels: candidate-4.12, candidate-4.13, fast-4.12)
No updates available. You may still upgrade to a specific release image with --to-image or wait for new updates to be available.
```

After the upgrade finishes:

```console
$ oc adm upgrade
Cluster version is 4.12.11

Upstream is unset, so the cluster will use an appropriate default.
Channel: fast-4.12 (available channels: candidate-4.12, candidate-4.13, fast-4.12)
No updates available. You may still upgrade to a specific release image with --to-image or wait for new updates to be available.
```

Reply:
Thanks a lot for the output @petr-muller 
I have one question. In the previous `oc get clusterverison` output, we are representing the versions in the `<4.y+2.z>` format so that the output is universal for all the versions. In the same way, I tried to update the `oc adm upgrade` output that you provided. Can you please confirm if that[1] makes sense? 

[1] `Cluster version is <4.y+2.z> Upstream is unset, so the cluster will use an appropriate default. Channel: fast-4.y+2 (available channels: candidate-4.y+2, candidate-4.y+3, fast-4.y+2) No updates available. You may still upgrade to a specific release image with --to-image or wait for new updates to be available.`

Reply:
I wonder if we can simply omit the irrelevant part of the output? There is some variance of what the output will show and it does not make sense to predict what exactly is shown. I'd say that our example should focus only on the first line and omit the rest of the output (that can vary greatly - there may be various channels and various recommended upgrades).

During upgrade:
```console
$ oc adm upgrade
info: An upgrade is in progress. Working towards <4.y+2.z>: 104 of 830 done (12% complete), waiting on etcd, kube-apiserver
...

After completed upgrade:
```
and 
```console
Cluster version is <4.y+2.z>
...
```

Please make sure you keep the newlines in the output (currently the PRs concatenated all lines from the original output into a single line) and make sure you also update the example on L94.

Reply:
@petr-muller Okay. Agreed. I updated the PR with the output[1]. I chose that output because users will be running the `oc adm upgrade` command once the upgrade is complete. I updated L94. Let me know if it looks good to you! Thanks a lot again :)

[1]
```
Cluster version is <4.y+2.z>
...
```